server/pages.js

```diff
@@ -189,10 +187,10 @@ pages.add(
 // ----------
 
 // New collective page - we keep the v2 alias because we shared some of these URLs by email
-pages.add('new-collective-page', '/:slug/v2');
+pages.add('new-collective-page', '/:slug');
 
 // Collective page
-pages.add('collective', '/:slug', 'new-collective-page');
```

Comment:
Maybe better to add a second route for the collective page with onboarding, so we keep this simple route: `pages.add('collective', '/:slug', 'new-collective-page');`

Reply:
Sounds good